### ISSUE TITLE
Kappa V4.0 T1, white screen. pls help

### ISSUE THREAD

**Issue by ghost** [rre36/ShaderpackIssueTracker#18]
My Minecraft screen turns white when i use kappa V4.0 T1, but V3.1 works fine.
![2021-03-29_12 20 52](https://user-images.githubusercontent.com/81144349/112793505-71db8d80-908b-11eb-8ad4-5a1db6aadbf4.png)

**Environment (please complete the following information):**
Optifine Version: 1.16.3 HD U G5
Minecraft Version: 1.16.3
Shaderpack Version: V4.0 T1
Graphics Card and Driver Version: NVIDIA Geforce GT 1030, Driver Version : 14.8.0.1042
Operating System: 64 bit
CPU: Intel(R) Core(TM) i5-7600 CPU @ 3.50Ghz

Log?
[latest.log](https://github.com/rre36/ShaderpackIssueTracker/files/6219652/latest.log)

**Comment by xandercreates** [comment]
Your Driver version is 456.71 which is outdated so that might be the issue. Also why are you on 1.16.3 if you dont mind me asking

**Comment by xandercreates** [comment]
tl;dr, Update your Graphics Drivers

**Comment by xandercreates** [comment]
https://www.nvidia.com/Download/driverResults.aspx/171474/en-us

**Comment by ghost** [comment]
> https://www.nvidia.com/Download/driverResults.aspx/171474/en-us

ty

**Comment by xandercreates** [comment]
no prob
